SonarRunner.Shim/SonarRunner.Wrapper.cs

```diff
@@ -61,18 +62,16 @@ public ProjectInfoAnalysisResult Execute(AnalysisConfig config, ILogger logger)
 
         #region Private methods
 
-        private static string FindRunnerExe(ILogger logger)
+        private static string FindRunnerExe(AnalysisConfig config, ILogger logger)
         {
-            string exeFileName = FileLocator.FindDefaultSonarRunnerExecutable();
-            if (exeFileName == null)
-            {
-                logger.LogError(Resources.ERR_FailedToLocateSonarRunner, FileLocator.SonarRunnerFileName);
-            }
-            else
-            {
-                logger.LogMessage(Resources.DIAG_LocatedSonarRunner, exeFileName);
-            }
-            return exeFileName;
+            // TODO This is a hack
+            var binFolder = Path.Combine(config.SonarConfigDir, @"..\bin");
```

Comment:
How about making the bin directory part of the AnalysisConfig? (there's no reason now to treat it differently from the config directory and the out directory - they are all created as part of the analysis).